docs/api/command-line-switches.md

```diff
@@ -67,13 +67,27 @@ Enables caller stack logging for the following APIs (filtering events):
 
 * `desktopCapturer.getSources()` / `desktop-capturer-get-sources`
 
-### --enable-logging
+### --enable-logging[=file]
 
-Prints Chromium's logging into console.
+Prints Chromium's logging to stderr (or a log file).
 
-This switch can not be used in `app.commandLine.appendSwitch` since it is parsed
-earlier than user's app is loaded, but you can set the `ELECTRON_ENABLE_LOGGING`
-environment variable to achieve the same effect.
+This switch can not be used in `app.commandLine.appendSwitch` since it is read
+before the first line JavaScript is run.
+
+The `ELECTRON_ENABLE_LOGGING` environment variable has the same effect as
+passing `--enable-logging`.
+
+If `destination` is:
+
+* an empty string: logs will be printed on stderr.
+* `file`: logs will be saved to the file specified by `--log-file=...`, or to
+  `electron_debug.log` in the user-data directory if `--log-file` is not
```

Comment:
Could we be nice here and do `{app_name}_debug.log` just so it's, idk, branded 😆

Reply:
child processes can't get the app name, so we'd need to add another command-line argument, which I don't think is worth it.